app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/assets/AssetsCommonViews.kt

```diff
@@ -48,6 +53,26 @@ fun Modifier.assetPlaceholder(
     )
 }
 
+fun Modifier.dashedCircle(size: Dp) = composed {
```

Comment:
Nice 👌 . Since you added such modifier, I think you achieved the same design as `ValidatorsSize` composable in Account screen. Could you replace that with your custom function if possible?